wg-outreach/meeting-notes/2019-04-29.md

```diff
@@ -0,0 +1,34 @@
+# Outreach WG
+April 29, 2019
+
+## Attendees
+**Members**
+- David (@cruzerld)
+- JD (@jdalton)
+- Sofia (@sofianguy)
+
+**Visitors**
+- Shelley (@codebytere)
+
+## Agenda
+* JD build electron on Windows 10:
+    * https://hackmd.io/s/rJ67Tjc5V
+    * all told, took days including troubleshooting
+* David build electron on Linux:
+    * https://hackmd.io/YJmFdvFrRYGfNgzW74EHSA 
+    * need to organize the build instructions order
+* Season of Docs follow up
+    * decision will be released on April 30, 2019 12:00 UTC 
+* Follow up on action items from [2019-04-01](https://github.com/electron/governance/blob/master/wg-outreach/meeting-notes/2019-04-01.md#action-items) meeting
+* Action Items [Project Board](https://github.com/electron/governance/projects/4) 
```

Comment:
Is the Project Board a new thing?

Reply:
Oops, I forgot to talk about that during the meeting. I made a project board to keep track of our ongoing Action Items. Sometimes we forget to revisit items each meeting.

Reply:
I'll delete that from this set of meeting notes and put it on next meeting's

Reply:
Ah okay!